services/docker-compose.devel.yml

```diff
@@ -63,6 +63,10 @@ services:
     volumes:
       - ./storage:/devel/services/storage
       - ../packages:/devel/packages
+    environment:
+      - SC_BOOT_MODE=debug-ptvsd-disable
```

Comment:
why the value assigned finishes with **disable** ??

Reply:
to disable it